### ISSUE TITLE
Consider ways to allow a user to permanently suppress the low resolution error

### ISSUE THREAD

**Issue by Norc** [foundryvtt/foundryvtt#14217]
### Current User Experience
Sometimes, a user must knowingly run Foundry VTT with resolution that repeatedly triggers the "Low Resolution" error:

https://user-images.githubusercontent.com/3605541/229001163-89b40c59-5d5d-4956-916c-2020084046cd.png

This can happen due to hardware limitations, and also due to accessibility issues. For users who this affect, this can be quite disruptive. For example, this was originally opened by #9144, before that issue was repurposed for a different fix. This is another example from a customer support email:

> As someone with low vision, I run my desktop at a lower than typical screen resolution (1080p 19x10), and have font scaling at 150% which makes the effective resolution even lower.[...]
> 
> Currently, Foundry (and in Forge) throws a red "...requires a usable window dimensions..." error which requires a click to dismiss, as if it were a critical failure. And it's not in one place, it's actually in mulitple places. Foundry start, world load, in-game when you change browser zoom levels.  and other triggers as well. THIS IS VERY ANNOYING. 
### Existing workarounds
Some modules have cropped up that help address this, notably https://foundryvtt.com/packages/custom-css, https://foundryvtt.com/packages/hide-player-ui, and https://foundryvtt.com/packages/warning-remove but none are currently updated for V14.
### Proposed solution
Create a new client-level core setting that converts this error to a console-only log statement

**Comment by GrumpyCornGames** [comment]
Agreed, this should be done, its an accessibility issue and a nuisance warning.
